Fix sepolicy for netd_integration_test

qshi5 opened the pull request:
Add missing sepolicy for NetdSELinuxTest#CheckProperMTULabels case

Test done: netd_integration_test64

Tracked-On: https://jira.devtools.intel.com/browse/OAM-129963

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

qshi5 committed:
Fix sepolicy for netd_integration_test

Add missing sepolicy for NetdSELinuxTest#CheckProperMTULabels case

Test done: netd_integration_test64

Tracked-On: https://jira.devtools.intel.com/browse/OAM-129963
Signed-off-by: Shi, Qiming <qiming.shi@intel.com>

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is SUCCESS. Please check the linked Tracked-On issue/Android CI Web  for more details. For Binaries: /cactus-absp-or-local/celadon-merge/1032
